assemblies/assembly_business-processes/main.adoc

```diff
@@ -13,7 +13,7 @@ include::_artifacts/author-group.adoc[]
 As a business rules and processes developer, you can use {CENTRAL} in {PRODUCT} to design business processes to meet specific business requirements. {PRODUCT} provides example projects with example business assets directly in {CENTRAL} as a reference. This document describes how to create and test an example mortgage application project based on the *Mortgage_Process* sample project included in {CENTRAL}.
 
 .Prerequisites
-* {EAP_LONG} 7.2.0 is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.2/html-single/installation_guide/[_Red Hat JBoss EAP 7.2.0 Installation Guide_].
+* {EAP_LONG} {EAP_VERSION} is installed. For details, see the https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html-single/installation_guide/[_{EAP_LONG} {EAP_VERSION} Installation Guide_].
```

Comment:
Should the URL version be /7.2/?

For sentences referring to EAP, which EAP version is preferable? 7.2 {EAP_VERSION} or 7.2.0 {EAP_VERSION_LONG}?